group_project/assignment_3.py

```diff
@@ -1,5 +1,8 @@
 def replace_bad_words(a_string, word_to_replace, new_word):
-    pass
+    if word_to_replace not in a_string:
+        return a_string
```

Comment:
There's no need to check if the word is not part of the string. If it's not, Python will just make no replacements whatsoever. Example: https://repl.it/Md4N

Reply:
you can just stick to what you did in assignement 2, just checking if the string is in a_string.